src/EditorFeatures/CSharpTest/ChangeSignature/RemoveParametersTests.cs

```diff
@@ -293,11 +293,11 @@ class C
 
                 var handler = new CSharpChangeSignatureCommandHandler(TestWaitIndicator.Default);
                 var delegatedToNext = false;
-                Func<CommandState> nextHandler = () =>
+                CommandState nextHandler()
                 {
                     delegatedToNext = true;
                     return CommandState.Unavailable;
-                };
+                }
```

Comment:
It can improve things in that the compiler is more capable of optimizing things.  For example, if the local function is invoked and not passed, we are open to the optimization of the state-capturing happening through a struct.